tests/sentry/snuba/metrics/test_extraction.py

```diff
@@ -34,73 +34,49 @@ def test_equality_of_specs(default_project) -> None:
     [
         ("count()", "release:a", False),  # supported by standard metrics
         ("failure_rate()", "release:a", False),  # supported by standard metrics
-        ("count_unique(geo.city)", "release:a", False),
```

Comment:
You can use the split view for this section so it can be easier to review.

In this section, I'm mostly flattening the tuples to be only a single line instead of four and putting the comments above the line.

There are some additions to this list which I will describe below with a comment.

<img width="207" alt="image" src="https://github.com/getsentry/sentry/assets/44410/4dad3655-0405-44a5-9db3-bad86e18c5fb">